payments/src/main/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidator.java

```diff
@@ -39,11 +39,7 @@ public void isAuthorize(String ptIdDominioReq, String ptIdIntermediarioReq, Stri
             gpdClient.getOrganization(ptIdDominioReq);
         }  catch (Exception e) {
             log.error("[isAuthorize ERROR] error during GPD call" , e);
-            if (e.getCause() instanceof FeignException.FeignClientException) {
-                throw new PartnerValidationException(PaaErrorEnum.PAA_ID_DOMINIO_ERRATO);
-            } else {
-                throw new PartnerValidationException(PaaErrorEnum.PAA_SYSTEM_ERROR);
-            }
+            throw new PartnerValidationException(PaaErrorEnum.PAA_ID_DOMINIO_ERRATO);
```

Comment:
why?

Reply:
according to this [workflow](https://pagopa.atlassian.net/wiki/spaces/PPD/pages/479987315/Controlli+payments)

Reply:
but if there is a network problem the PAA_ID_DOMINIO_ERRATO it is not appropriate

Reply:
I agree with you...we need to modify again the workfow :)